pinax/stripe/actions/events.py

```diff
@@ -30,14 +33,16 @@ def add_event(stripe_id, kind, livemode, message, api_version="", request_id="",
         webhook.process()
 
 
-def dupe_event_exists(stripe_id):
+def dupe_event_exists(stripe_id, stripe_account=None):
```

Comment:
I believe Stripe are using globally unique IDs for all their resources, so passing in the stripe_account has no effect in this function's case. I think it makes sense to keep this lookup limited to `stripe_id` for other reasons too:

- If an event exists in the database with a `stripe_account` of non-null and legacy code calls this method, without specifying `stripe_account`, the lookup will fail.
- If someone upgrades pinax-stripe adding the `stripe_account` column to their schema, all values for this column will be null. If they're a keener and begin specifying a value for `stripe_account` the lookup will fail.

Reply:
It makes sense. @blueyed do you agree ?

Reply:
Yes, will remove it.